src/richie/apps/courses/templates/courses/cms/person_detail.html

```diff
@@ -1,6 +1,17 @@
 {% extends "richie/fullwidth.html" %}
 {% load cms_tags extra_tags i18n thumbnail %}
 
+{% block meta_opengraph_contextuals %}
+    <meta property="og:title" content="{{ current_page.get_title|truncatechars:65 }}">
+    <meta property="og:url" content="{{ SITE.web_url }}{{ current_page.get_absolute_url }}">
+    {% comment %}<meta property="og:type" content="profile">
+    <meta property="profile:first_name" content="Should be filled with person firstname">
+    <meta property="profile:last_name" content="Should be filled with person lastname">
```

Comment:
we don't differentiate firstname and lastname anymore. Is there a more general "name" property?

Reply:
Nope, there is not unified og tag for full name. All in all, this is not really important i think, we didn't even know if we have right to promote person profile like this, so i will just remove it.